src/UI/Component/Input/Field/Factory.php

```diff
@@ -344,5 +344,37 @@ public function password($label, $byline = null);
 	 * @return \ILIAS\UI\Component\Input\Field\Select
 	 */
 	public function select($label, array $options, $byline = null);
+
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     A radio-input is used to depict a choice of options excluding each other.
+	 *   composition: >
+	 *     The radio is considered as one field with a label and a number of
+	 *     options. Each option in turn bears a label in form of a positive statement.
+	 *   effect: >
+	 *     If used in a form, each option of a radio may open a dependant section (formerly known
+	 *     as sub form).
+	 *   rivals:
+	 *     Checkbox field: Use a checkbox field for a binary yes/no choice.
+	 *     Select: >
+	 *       Use selects to choose items from a longer list as the configuration of
+	 *       an aspect; when the choice has severe effects on, i.e. service behavior,
+	 *       or needs further cconfiguration, stick to radios.
+	 *
+	 * rules:
+	 *   wording:
+	 *     1: The options' labels MUST always state something positive.
+	 *
```

Comment:
I have some more:

usage:

- A Radio Input SHOULD contain 3 to 5 options. If there are more, the Select Input might be the better option.
- They MAY also be used to select between two options where one is not automatically the inverse of the other (such as “Order by Date” and “Order by Name”).

wording:
1: Good, but give an example to make the point really clear here.
- Each option in a Radio Input MUST be labeled. 
 - This label SHOULD not consist of more than 5 words. 
- The label of the options SHOULD not simply repeat the label of the Radio. A meaningful labeling SHOULD be chosen instead.

Ordering:
- The presumably most relevant option SHOULD be the first option.